modules/contacts/modules.php

```diff
@@ -106,7 +131,17 @@ protected function output() {
  */
 class Hm_Output_contacts_content_start extends Hm_Output_Module {
     protected function output() {
-        return '<div class="contacts_content"><div class="content_title">'.$this->trans('Contacts').'</div>';
+        $contact_source_list = $this->get('contact_sources', array());
+        $actions = '<div class="src_title">'.$this->html_safe('Export Contacts as CSV').'</div>';
+        $actions .= '<div class="list_src"><a href="?page=export_contact&amp;contact_source=all">'.$this->html_safe('All Contacts').'</a></div>';
+        foreach ($contact_source_list as $value) {
+            $actions .= '<div class="list_src"><a href="?page=export_contact&amp;contact_source='.$this->html_safe($value).'">'.$this->html_safe($this->html_safe($value).' Contacts').'</a></div>';
```

Comment:
html_safe only needed on the value variable. Please translate the static strings.